core/market-forwarding/Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2018"
 
 [dependencies]
 ya-client = "0.3"
+ya-client-model = "0.1"
```

Comment:
not required, since it is visible via `ya_client::model`

Reply:
OK, I will update this for all crates.